### DISCUSSION TITLE
Chatwoot for free?

### DISCUSSION THREAD

**Discussion by guiaguiar27** [chatwoot/chatwoot#9854] [General]
I would like to know since chatwoot is an open source platform. When it's installed in a VPS I still need to pay for the amount of agents that I want to have?

**Comment by sojan-official** [comment]
Chatwoot is available in both community and enterprise version 
ref: https://www.chatwoot.com/hc/user-guide/articles/1677776492-enterprise-edition 

there is no cost for using the community edition.
